configs/distill/mmcls/dist/dist_logits_resnet34_resnet18_8xb32_in1k.py

```diff
@@ -0,0 +1,43 @@
+_base_ = [
+    'mmcls::_base_/datasets/imagenet_bs32.py',
+    'mmcls::_base_/schedules/imagenet_bs256.py',
+    'mmcls::_base_/default_runtime.py'
+]
+
+teacher_ckpt = 'https://download.openmmlab.com/mmclassification/v0/resnet/resnet34_8xb32_in1k_20210831-f257d4e6.pth'  # noqa: E501
+
+model = dict(
+    _scope_='mmrazor',
+    type='SingleTeacherDistill',
+    data_preprocessor=dict(
+        type='ImgDataPreprocessor',
```

Comment:
I'm not sure whether we have to use `mmcls.ImgDataPreprocessor`.

Reply:
YES，it is used to prepare input images

Reply:
I mean whether we have to use `mmcls.ImgDataPreprocessor` but not `ImgDataPreprocessor`. As the default scope is mmrazor, if we want to use ImgDataPreprocessor in mmcls, maybe we have to add `mmcls.` prefix. But I'm not sure.

Reply:
"mmcls." is not necessary, the config is  excutable.